src/Microsoft.DotNet.Interactive/Connection/KernelCommandAndEventSenderExtensions.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.DotNet.Interactive.Events;
+
+namespace Microsoft.DotNet.Interactive.Connection
+{
+    public static class KernelCommandAndEventSenderExtensions
```

Comment:
Neither of these convenience methods should be public.

Reply:
done